sqlglot/optimizer/eliminate_join_marks.py

```diff
@@ -0,0 +1,184 @@
+import typing as t
+import copy
+
+from sqlglot import expressions as exp
+from sqlglot.optimizer.scope import build_scope, find_all_in_scope
+
+
+def _update_from(
```

Comment:
can you move the public method to the top, and all the private methods to the bottom

Reply:
moved